docs/roadmap.rst

```diff
@@ -0,0 +1,72 @@
+.. include:: links.rst
+
+--------------------
+Development road map
+--------------------
+
+This road-map serves as a guide for developers as well as a way for us to
+communicate to users and other stake-holders aboout the expectations they should
+have about the current functionality of the software and future developments.
+
+Version 0.3 (Targetted for March 1st, 2020)
+-------------------------------------------
+This version should be considered an early alpha of the software, but will
+contain a full pipeline of processing from a raw BIDS dataset to analyzable data.
+
+At this point, the processing pipeline will include the following major steps:
+
+#. Susceptibility distortion correction.
+    Using SDC flows
+
+Version 0.4 (April 1st, 2020)
+-----------------------------
+#. Head motion correction.
+
+#. Eddy current correction.
+
+    For this release, these two will be done doing something related to the
+    SHOREline approach In cases where the data are "shelled", 3dSHORE will be
+    used as the diffusion model. If the data are single-shell, we will use SFM
+    as the diffusion model.
+
+#. Identification of outliers.
+
+Version 0.5 (May 1st, 2020)
+----------------------------
+
+#. Registration between dMRI and T1w image.
+
+#. Identification of outlier measurements (+ imputation?)
```

Comment:
is this outlier in the sense of head motion? we should be more clear about that

Reply:
That, but also other artifacts (e.g., dropped slices, zippers and so on). I've tried to clarify in 41ee304